eos-packagelist/eos-packagelist

```diff
@@ -107,6 +107,7 @@ if __name__ == '__main__':
 
     # Get the data from the internet and parse it
     blacklist = []
+    netinstall_packages = []
     if (args.arch and args.arch != "x86_64") or (not args.arch and platform.machine() != "x86_64"):
         try:
             url = "https://raw.githubusercontent.com/endeavouros-arm/install-script/master/blacklist"
```

Comment:
The blacklist is mainly for Desktop-Base list which isn't there in the netinstall.yaml file in ARM.
So you've found the first bug since 
```
 eos-packagelist "Desktop-Base + Common packages"   
```
Shows an error. I will try to fix it and get back to you

Reply:
I will work with Pudge on revising the blacklist too